--- ../src/clibio.c	2023-01-06 21:53:02.114368373 -0600
+++ clibio.c	2023-01-06 22:23:52.462777156 -0600
@@ -153,7 +153,6 @@
 #endif
 	DCX	H
 	SPHL			/* Set up stack */
-#ifndef HDOS			 /* Parse command line; build args from $AS */
 #ifndef NOMAC
 	LXI	D,Q8QENDD	/* Set sbrk locn to higher of code, data segs */
 	PUSH	D
@@ -162,8 +161,10 @@
 	POP	H
 	JZ	$A81
 	SHLD	$LM
+$A81:
 #endif
-$A81:	LXI	H,ARGLIN	/* Get address of arg line */
+#ifndef HDOS			 /* Parse command line; build args from $AS */
+	LXI	H,ARGLIN	/* Get address of arg line */
 	MOV	E,M		/* And char count */
 	MVI	M,' '		/* Add on space we might or might not take */
 	MVI	D,0
